File: Modul_4/Bahasa_C/PRAK401-2510817120005-NorKhalisyah.c
#include <stdio.h>
int main () {
    int n;
    char simbol;
    
    printf("Masukkan nilai dan simbol kelipatan: ");
    scanf("%d %c", &n, &simbol);

    for (int i = 1; i <= 50; i++) {
        if (i % n == 0)
            printf("%c ", simbol);
        else
            printf("%d ", i);
        
        if (i % 25 == 0)
            printf("\n");
    }

    return 0;
}

File: Modul_4/Bahasa_Python/PRAK401-2510817120005-NorKhalisyah.py
n, simbol = input("Masukkan nilai dan simbol kelipatan: ").split()
n = int(n)

for i in range(1, 51):
    if i % n == 0:
        print(simbol, end=" ")
        
    else:
        print(i, end=" ")
    
    if i % 25 == 0:
        print()